# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-06-08 09:18-0400\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"

#: plugins/kickstart.py:60
msgid "Install packages defined in a kickstart file on your system"
msgstr ""

#: plugins/kickstart.py:65
msgid "kickstart file"
msgstr ""

#: plugins/kickstart.py:83
#, python-format
msgid "file cannot be parsed: %s"
msgstr ""

#: plugins/rpm_conf.py:65
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr ""

#: plugins/rpm_conf.py:71
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr ""

#: plugins/rpm_conf.py:84
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr ""

#: plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr ""

#: plugins/showvars.py:46
msgid "show all active dnf variables"
msgstr ""

#: plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr ""

#: plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr ""

#: plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr ""

#: plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr ""

#: plugins/snapper.py:74
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""

#: plugins/snapper.py:80
msgid "creating post_snapshot"
msgstr ""

#: plugins/snapper.py:85
#, python-format
msgid "created post_snapshot %d"
msgstr ""

#: plugins/snapper.py:89
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr ""

#. Translators: This string is only used in unit tests.
#: plugins/system_upgrade.py:46
msgid "the color of the sky"
msgstr ""

#: plugins/system_upgrade.py:62
msgid "Need a --releasever greater than the current system version."
msgstr ""

#. Translators: do not change "reboot" here
#: plugins/system_upgrade.py:64
#, python-brace-format
msgid ""
"Download complete! Use 'dnf {command} reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf {command} clean'"
msgstr ""

#: plugins/system_upgrade.py:67
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""

#: plugins/system_upgrade.py:122
#, python-format
msgid "Screen blanking can't be disabled: %s"
msgstr ""

#: plugins/system_upgrade.py:143
#, python-format
msgid "Failed loading state file: %s, continuing with empty state."
msgstr ""

#: plugins/system_upgrade.py:295
msgid "The following boots appear to contain upgrade logs:"
msgstr ""

#: plugins/system_upgrade.py:305
msgid "-- no logs were found --"
msgstr ""

#: plugins/system_upgrade.py:320
msgid "Cannot find logs with this index."
msgstr ""

#: plugins/system_upgrade.py:329
msgid "Unable to match systemd journal entry"
msgstr ""

#: plugins/system_upgrade.py:350
msgid "Prepare system for upgrade to a new release"
msgstr ""

#: plugins/system_upgrade.py:360
msgid "keep installed packages if the new release's version is older"
msgstr ""

#: plugins/system_upgrade.py:364
msgid "which logs to show"
msgstr ""

#: plugins/system_upgrade.py:398
#, python-brace-format
msgid "Incompatible version of data. Rerun 'dnf {command} download [OPTIONS]'"
msgstr ""

#: plugins/system_upgrade.py:435
msgid "Command 'offline-distrosync' cannot be used with --no-downgrade option"
msgstr ""

#: plugins/system_upgrade.py:457
msgid "Additional information for System Upgrade: {}"
msgstr ""

#: plugins/system_upgrade.py:460
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""

#: plugins/system_upgrade.py:464
msgid "Operation aborted."
msgstr ""

#: plugins/system_upgrade.py:527
msgid "system is not ready for upgrade"
msgstr ""

#: plugins/system_upgrade.py:530
#, python-brace-format
msgid ""
"the transaction was not prepared for '{command}'. Rerun 'dnf {command} "
"download [OPTIONS]'"
msgstr ""

#: plugins/system_upgrade.py:534
msgid "upgrade is already scheduled"
msgstr ""

#: plugins/system_upgrade.py:540
msgid "trigger file does not exist. exiting quietly."
msgstr ""

#: plugins/system_upgrade.py:543
msgid "another upgrade tool is running. exiting quietly."
msgstr ""

#: plugins/system_upgrade.py:552
#, python-brace-format
msgid "use 'dnf {command} reboot' to begin the upgrade"
msgstr ""

#: plugins/system_upgrade.py:570
msgid "Rebooting to perform upgrade."
msgstr ""

#: plugins/system_upgrade.py:594
msgid "Starting offline upgrade. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:596
msgid "Starting offline distrosync. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:598
msgid "Starting system upgrade. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:630 plugins/system_upgrade.py:639
#, python-format
msgid "Unable to match package: %s"
msgstr ""

#: plugins/system_upgrade.py:644
msgid "Unable to match some of packages"
msgstr ""

#: plugins/system_upgrade.py:647
msgid "Cleaning up downloaded data..."
msgstr ""

#: plugins/system_upgrade.py:725
msgid "Download finished."
msgstr ""

#: plugins/system_upgrade.py:729 plugins/system_upgrade.py:730
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr ""

#: plugins/system_upgrade.py:739
msgid "Prepare offline upgrade of the system"
msgstr ""

#: plugins/system_upgrade.py:744
msgid "Prepare offline distrosync of the system"
msgstr ""

#: plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""

#: plugins/tracer.py:70
msgid "For more information run:"
msgstr ""

#: plugins/tracer.py:97
msgid "Call to Tracer API ended unexpectedly:"
msgstr ""

#: plugins/tracer.py:99
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""

#: plugins/tracer.py:101
msgid "We apologize for any inconvenience"
msgstr ""
